instrumentation/spring-webflux-5.0/spring-webflux-5.0.gradle

```diff
@@ -30,8 +30,7 @@ testSets {
 }
 
 dependencies {
-  compileOnly group: 'org.springframework', name: 'spring-webflux', version: '5.0.0.RELEASE'
-  compileOnly group: 'io.projectreactor.ipc', name: 'reactor-netty', version: '0.7.0.RELEASE'
```

Comment:
i think it makes sense to keep these, since these classes are used directly in this module (even though they also come in as transitive dependencies now)